modules/codegen/src/main/kotlin/jp/co/soramitsu/iroha2/parse/SchemaReader.kt

```diff
@@ -16,7 +17,7 @@ class SchemaReader {
     fun readSchema(fileName: String): Map<String, Any> {
         val resource = Thread.currentThread().contextClassLoader.getResourceAsStream(fileName)!!
         val sb = StringBuilder()
-        val lines = resource.bufferedReader().lines().toList()
+        val lines = resource.bufferedReader().lines().collect(Collectors.toList<String>())
```

Comment:
smells like Java, why so?

Reply:
hm, I don't remember why I would change this, restored now